2-mandatory/2-water-bottle.js

```diff
@@ -19,20 +19,37 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    if(this.volume === 0){
+      return this.volume += 100
+    } else { return this.volume}
+    
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    if(this.volume <= 90 && this.volume >= 0){
+      return this.volume +=10
+    }
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    if(this.volume <= 100 && this.volume >= 10){
+      return this.volume -=10
+    }
   },
   isFull: function () {
     // this function should return true if your bottle is full;
+    if(this.volume === 100){
+      return true
+    } else return false
```

Comment:
Nice one!

This can also be written as:
```suggestion
    return this.volume === 100;
```
Does that make sense why?

Reply:
Same below